packages/react/src/themes/teams/components/Toolbar/toolbarVariables.ts

```diff
@@ -46,6 +53,26 @@ export interface ToolbarVariables {
   borderWidth: string
   borderRadius: string
   dividerMargin: string
+
+  menuWidth: string
+  menuPadding: string
+  menuBackground: string
+  menuBorder: string // border color
+  menuBorderWidth: string
+  menuBorderRadius: string
+  menuBoxShadow: string
+
+  menuItemForeground: string
+  menuItemForegroundHover: string
+  menuItemBackgroundHover: string
+  menuItemBackgroundFocus: string
+  menuItemForegroundFocus: string
+  menuItemForegroundDisabled: string
+  menuItemBackgroundDisabled: string
```

Comment:
As I remember we use following naming: `PROP + SLOT + VARNAME`

Reply:
In color scheme we have `foregroundDisabled`. Here I just prefixed it with `SLOT`. Let's discuss this with @mnajdova separately.